processFolder.py

```diff
@@ -0,0 +1,56 @@
+# Tayo Elelu
+# Created 10/14/16
+# CS 4365 - Software Engr II
+# Image Processing Module
+
+# Using Pillow library
+# Given a directory, this module will analyze the folder.
+# It will go thru all images, resize
+# them to 75x75 convert them to grayscale,
+# and save the images for later use/output an array of them.
+
+################################################################
+
+#Import needed Pillow libraries
+from PIL import Image # image libraries and operations
+import os, sys        # directories 
+import imghdr         # image file recognition
+import numpy          # numpy arrays
+
+# Sample Paths
+# /mnt/c/Users/Owner/Pictures/SamplePictures/
+# /mnt/c/Users/Tayo/Documents/SE2CS4365/TestImages/
+
+path = raw_input("Enter the name of the directory you want to manipulate. \n")
+dirs = os.listdir(path) # store path
+print("Start.")
+
+#list of image types 
+imgfiles = ['rgb','gif','pbm','pgm','ppm','tiff','rast','xbm','jpeg','jpg','bmp','png','webp','exr']

```

Comment:
Hmm so we want to have the user enter the types of images to consider? I personally think it would be better to put it in with the module - types of files the user wants for the dataset will always be images, and shouldn't change with user input. Unless we want the user to specify some image types, and not others, which gets into another problem